pepper-apis/src/main/java/org/broadinstitute/ddp/filter/UserAuthCheckFilter.java

```diff
@@ -115,27 +114,33 @@ private void handleTemporaryUser(Request request) {
 
         if (!inWhitelist) {
             LOG.warn("Request '{} {}' is not in temp-user whitelist", requestedMethod, requestedPath);
-            throw halt(401);
+            throw ResponseUtil.haltError(HttpStatus.SC_UNAUTHORIZED,
+                    new ApiError(ErrorCodes.AUTH_CANNOT_BE_DETERMINED, "Request is not in temp-user whitelist"));
```

Comment:
Do we prefer error code to be "OPERATION_NOT_ALLOWED" rather than AUTH_CANNOT_BE_DETERMINED !!

Reply:
Maybe I'm being too paranoid, but can we make all the scenarios return a general body like this one?
```
{
  "code": "UNAUTHORIZED",
  "message": "Unauthorized"
}
```

Reply:
Happy to be convinced otherwise though.